modules/ztp-pgt-config-best-practices.adoc

```diff
@@ -15,3 +15,10 @@ Consider the following best practices when customizing site configuration `{poli
 * `MachineConfig` CRs should be included as `extraManifests` in the `SiteConfig` CR so that they are applied during installation. This can reduce the overall time taken until the cluster is ready to deploy applications.
 
 * `{policy-gen-cr}` CRs should override the channel field to explicitly identify the desired version. This ensures that changes in the source CR during upgrades does not update the generated subscription.
+
+* The default setting for `policyDefaults.consolidateManifests` is `true`. This is the recommended setting for DU profile. Setting it to `false` could impact large scale deployments.
+
+* The default setting for `policyDefaults.orderPolicies` is `false`. This is the recommended setting for DU profile.  
+After the cluster installation is complete and a cluster becomes `Ready`, {cgu-operator} creates a `ClusterGroupUpgrade` CR corresponding to this cluster. 
+The `ClusterGroupUpgrade` CR contains a list of ordered policies defined by the `ran.openshift.io/ztp-deploy-wave` annotation.
+If you use the `{policy-gen-cr}` CR to change the order of the policies, this can create conflicts and result in failure to apply the configuration.
```

Comment:
```suggestion
If you use the `{policy-gen-cr}` CR to change the order of the policies, conflicts might occur and the configuration might not be applied.
```
Rewording to avoid an ambiguous pronoun reference ("this"). Reference: https://www.ibm.com/docs/en/ibm-style?topic=grammar-pronouns#ambiguous-pronoun-references